app/src/main/shared/components/Alert.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+
+interface AlertState {
+    show: boolean;
+    cssClass: string;
```

Comment:
Can `cssClass` not just be derived from props in `render`? I don't think it needs to be in the state.